moq-loc/draft-mzanaty-moq-loc.md

```diff
@@ -284,6 +287,238 @@ with the header data as additional data input.
+as shown below:
+
+~~~psuedocode
+codec: "avc3.42E01E"
+bitrate: 1000000
+framerate: 30
+width: 1280
+height: 720
+~~~
+
+When ready for publishing, each encoded video chunk is considered as input
+to MOQT Object payload. If encrypted, the output of encryption will serve as
+the object's payload. The `GroupID` is incremented by 1 at IDR Frame boundaries. The `ObjectID` is increment by 1 for each encoded video frame, starting at 0 and resetting to 0 at the start of a new group. The first encoded video frame, MOQT Object with `ObjectID` 0, shall be the IDR Frame and the rest of the encoded video frames corresponds to dependent frames (delta frames) and organized in the decode order.
+
+When mapping to QUIC for sending, one unidirectional QUIC stream is setup to deliver all the encoded video chunks within a MOQT group.
+
+When decoding at the 'End Consumer', the objects from each of the QUIC
+streams are fed in the ObjectID order to the decoder setup for the
+video track.
+
+
+## Application with single video track with temporal layers {#app-2-temp-video}
+
+This section describes an media application with a single video track having
+2 temporal layers.
+
+Below shows sample catalog properties (see {{profile}} and {{video-ext}}) for describing H.264 video with 2 temporal qualities at 15fps and 30fps.
+
+~~~psuedocode
+
+Video Track 1: 720p 30/60fps H.264 video
+codec: "avc3.42E01E"
+bitrate: 1500000
+framerate: 60
+width: 1280
+height: 720
+~~~
+
+
+When ready for publishing, each encoded video chunk is considered as input
+to MOQT Object payload. If encrypted, the output of encryption will serve as
+the object's payload. The `GroupID` is incremented by 1 at IDR Frame boundaries. Each MOQT group shall contain 2 SubGroups corresponding
+to the 2 temporal layers as shown below:
+
+~~~ psuedocode
+Layer:0/30fps Subgroup: 0 ObjectID: even
+Layer:1/60fps Subgroup: 1 ObjectID: odd
+~~~
+
+Within the MOQT group, `ObjectID` is increment by 1 for each encoded video frame, starting at 0 and resetting to 0 at the start of a new group. The first encoded video frame, MOQT Object with `ObjectID` 0, shall be the IDR Frame and the rest of the encoded video frames corresponds to dependent frames (delta frames) and organized in the decode order. When mapping to
+QUIC for sending, one unidirectional QUIC stream is used per SubGroup,
+thus resulting in 2 QUIC streams per MOQT group.
+
+When decoding at the 'End Consumer' for a given MOQT group, the objects
+must be fed in the ObjectID order. This implies that the consumer
+media application needs to order objects across the SubGroup QUIC
+streams.
+
+
+
+## Application with mutiple dependant video tracks
+
+This section describes an media application with multiple dependent spatial tracks but all with same framerate.
+
+Below shows sample catalog properties (see {{profile}} and {{video-ext}}) for describing H.264 video with 2 temporal qualities at 15fps and 30fps.
```

Comment:
2 temporal qualities at 15fps and 30fps -> 2 spatial qualities at 360p and 720p each at 30 fps.
Same comment as above to combine above 2 sentences and use a standard template.